python/holidays/hr-HR.py

```diff
@@ -9,12 +9,13 @@ class hr_HR(Locale):
     01-01: [NF] Nova Godina
     01-06: [NRF] Bogojavljanje / Sveta tri kralja
     05-01: [NF] Praznik rada
+    05-30: [NF] Dan državnosti   
```

Comment:
Is the date of this holiday wrong or does it change from year to year (i.e. is not fixed)? If it does change: what is the algorithm to determine its date?